src/app/home-page/new-dashboard/new-dashboard.component.html

```diff
@@ -0,0 +1,55 @@
+<div fxLayout>
+  <app-my-sidebar fxFlexAlign="stretch"></app-my-sidebar>
+
+  <div id="row-holder" fxLayout="column" fxLayoutAlign="start start" class="p-4 w-100">
+    <mat-card class="alert alert-info mx-1 w-100" role="alert"> Major Release: </mat-card>
+    <div id="entry-holder" fxLayout="row" fxLayoutGap="2.5rem" class="w-100 mb-5 fx-grow">
+      <app-entry-boxes class="fx-grow box-container" entryType="Workflow"></app-entry-boxes>
+      <app-entry-boxes class="fx-grow box-container" entryType="Tool"></app-entry-boxes>
+      <app-entry-boxes class="fx-grow box-container" entryType="Service"></app-entry-boxes>
+    </div>
+    <div id="updates-holder" fxLayout="row" fxLayoutGap="2.5rem" class="w-100 fx-grow">
+      <mat-card class="fx-grow">
+        <div id="header-bar" fxLayout="row" fxLayoutGap="5px" class="mat-card-header-separate">
+          <div id="workflow-number" fxLayout="row">
+            <mat-card-header fxLayout="row">
+              <mat-card-title>Starred</mat-card-title>
+            </mat-card-header>
+          </div>
+        </div>
+        <mat-divider></mat-divider>
+        <mat-card-content class="p-3">
+          <span class="weight-bold">Recent Activity</span>
+        </mat-card-content>
+      </mat-card>
+      <mat-card class="fx-grow">
+        <div id="header-bar" fxLayout="row" fxLayoutGap="5px" class="mat-card-header-separate">
```

Comment:
Duplicate id; should be avoided, although not fatal: https://softwareengineering.stackexchange.com/questions/127178/two-html-elements-with-same-id-attribute-how-bad-is-it-really

Reply:
this will be changed and renamed when the starred/organization widgets are going to be made in the new ticket https://ucsc-cgl.atlassian.net/browse/SEAB-4623